featurebyte/models/tile.py

```diff
@@ -113,3 +113,36 @@ def check_time_modulo_and_frequency_minute(cls, values: Dict[str, Any]) -> Dict[
             )
 
         return values
+
+
+class TileCommonParameters(FeatureByteBaseModel):
+    """
+    Model for common parameters used by various steps within a tile scheduled job
+    """
+
+    tile_id: str
+    aggregation_id: str
+    tile_modulo_frequency_second: int
+    blind_spot_second: int
+    frequency_minute: int
+
+    sql: str
+    entity_column_names: List[str]
+    value_column_names: List[str]
+    value_column_types: List[str]
+
+    class Config:
+        """Model configuration"""
```

Comment:
May need to inherit `FeatureByteBaseModel.Config` if `validate_assignment` is desired.